src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -506,4 +533,34 @@ private MetaData createMetadata(String jurisdictionId, String caseTypeId, Map<St
 
         return metadata;
     }
+
+    private void validateSearchRequest(String searchRequest) throws IOException {
```

Comment:
getQuery() method is called within both the methods validateSearchRequestContainsQuery() and rejectBlackListedQuery(). As getQuery method converts JSON string to map, it may be good to convert it prior to the call to these two methods and pass the value as parameter.

Reply:
done